packages/protocol/lib/compatibility/ast-code.ts

```diff
@@ -213,17 +212,6 @@ export class MethodMutabilityChange extends MethodValueChange {
   }
 }
 
-/**
- * The input parameters of a method changed. Since the input parameters
- * are used as the id of a method, this should probably never appear.
- */
-export class MethodParametersChange extends MethodValueChange {
```

Comment:
are these the changes to make `public` --> `external` a non-major change? I'm guessing it we got off the abi....`public` --> `internal` will mark a major change

Reply:
Yes. `public` -> `internal` will still be caught because once the function is `internal` it won't even be seen by this code, so the checker will report a `MethodRemovedChange` (which is a major change).